ansible/roles/synapse-extra-config/templates/cm-extraconfig.yml

```diff
@@ -1,3 +1,4 @@
+# yamllint disable-file
```

Comment:
Ansible linter should not modify anything in templates folders
Not sure that the comment "# yamllint disable-file" will not mess things up in the kubernetes manifest